### ISSUE TITLE
php.lang.security.injection.tainted-sql-string does not detect SQL statement with newline

### ISSUE THREAD

**Issue by Sjord** [returntocorp/sgrep-rules#3376]
[php.lang.security.injection.tainted-sql-string](https://github.com/semgrep/semgrep-rules/blob/develop/php/lang/security/injection/tainted-sql-string.yaml)

```php
<?php
// Not detected
mysqli_query($handle, "
SELECT * FROM sometable WHERE field=".$_GET['input']);

// Detected
mysqli_query($handle, 
"SELECT * FROM sometable WHERE field=".$_GET['input']);
```
